@here/harp-datasource-protocol/lib/StyleSetEvaluator.ts

```diff
@@ -693,7 +693,7 @@ export class StyleSetEvaluator {
                 }
             }
 
-            if (typeof maxZoomLevel === "number" && zoomLevel > maxZoomLevel) {
```

Comment:
We need to document this change in BaseTechniqueParams in @here/harp-datasource-protocol/lib/TechniqueParams.ts.

Reply:
done